packages/story-editor/src/components/checklist/karma/prepublishSelect.karma.js

```diff
@@ -259,11 +259,14 @@ describe('Pre-publish checklist select offending elements onClick', () => {
           width: 640 / 2,
           height: 529 / 2,
           resource: {
-            width: 640,
-            height: 529,
+            id: 10,
             type: 'image',
             mimeType: 'image/jpg',
             src: 'http://localhost:9876/__static__/earth.jpg',
+            alt: 'earth',
```

Comment:
Sorry, my earlier comment was for _this_ alt text. we need this to be missing for the test to pass because we want to flag the missing alt text. 

```suggestion
            alt: '',
```